src/OTSession.js

```diff
@@ -70,11 +71,11 @@ export default class OTSession extends Component {
     });
   }
   disconnectSession() {
-    OT.disconnectSession((disconnectError) => {
+    OT.disconnectSession(this.props.sessionId, (disconnectError) => {
       if (disconnectError) {
         this.otrnEventHandler(disconnectError);
       } else {
-        const events = sanitizeSessionEvents(this.props.eventHandlers);
+        const events = sanitizeSessionEvents(this.props.sessionId, this.props.eventHandlers);
```

Comment:
Is this prop always going to be defined?

Reply:
Is there a case where we use the OTSession object without the sessionId? It's a required prop, right?